test/test_samples_osw/base/results_output.csv

```diff
@@ -1,237 +1,237 @@
-project_national-2945.osw,20.0,338.6,5.5,0.0,33.4,18.7,42.9,4742.0,0.0,0.0,40.3,0.0,1277.5,133.9,0.0,0.0,26.8,0.0,691.5,682.7,0.0,491.2,2045.2,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,480.4,0.0,0.0,0.0,610.0,1220.0,1220.0,0.0,0.0,0.0,41.0,13449.6,12229.0,0.0,true,15.2,15.2,0.0,0.0,158.9,0.0,0.0,0.0,95.3,0.0,0.0,0.0,0.0,682.0,45.9,41.7,30.0,10722.9,65.0,0.0,284.5,0.0,0.0,1649.4,1734.2,0.0,146.3,0.0
-project_national-2958.osw,20.0,469.0,11.0,111.1,43.7,23.0,42.9,3717.3,44.6,0.0,184.5,480.4,0.0,1081.0,0.0,0.0,85.4,0.0,0.0,0.0,0.0,1938.9,1658.3,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,480.4,0.0,0.0,0.0,845.0,1690.0,1690.0,0.0,0.0,15.0,0.0,13550.9,30455.4,0.0,true,19.8,15.2,0.0,0.0,1459.5,0.0,0.0,0.0,119.5,0.0,0.0,0.0,0.0,944.7,46.2,103.9,40.0,9913.3,195.2,0.0,1614.0,0.0,0.0,1941.3,2058.7,970.7,230.1,0.0
-project_national-2969.osw,20.0,0.0,5.5,147.1,0.0,0.0,45.5,955.4,0.0,0.0,142.7,0.0,343.8,129.6,0.0,0.0,94.9,0.0,0.0,0.0,0.0,1220.4,1231.3,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.3,0.0,45.4,0.0,0.0,853.0,853.0,0.0,0.0,43.0,180.0,2957.3,3283.7,0.0,true,0.0,16.1,0.0,0.0,174.9,0.0,0.0,0.0,96.2,0.0,0.0,0.0,0.0,0.0,10.1,11.2,30.0,4806.7,45.1,0.0,287.3,0.0,0.0,174.2,174.2,0.0,52.3,0.0
-project_national-2971.osw,20.0,390.4,11.0,73.9,1132.7,47.2,87.9,897.7,0.0,0.0,26.8,0.0,330.2,161.2,0.0,0.0,41.9,0.0,0.0,0.0,0.0,375.2,2259.5,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.3,0.0,0.0,383.5,1220.0,1220.0,1220.0,0.0,0.0,0.0,0.0,8754.4,9415.8,0.0,true,0.0,31.3,0.0,0.0,221.3,0.0,0.0,0.0,176.4,0.0,0.0,0.0,0.0,1364.0,29.9,32.1,40.0,6274.1,64.3,0.0,429.0,0.0,0.0,1166.3,1335.8,0.0,171.3,0.0
-project_national-2988.osw,20.0,390.4,11.0,30.6,66.6,35.1,409.5,2500.4,68.0,0.0,161.0,0.0,684.3,339.5,0.0,0.0,46.8,0.0,0.0,0.0,0.0,1535.4,1790.8,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.3,0.0,0.0,0.0,1220.0,1220.0,1220.0,0.0,0.0,0.0,0.0,14439.6,17436.5,0.0,true,30.2,0.0,0.0,0.0,458.4,0.0,0.0,0.0,176.0,0.0,0.0,0.0,0.0,1364.0,49.3,59.5,40.0,8124.5,94.2,0.0,664.6,0.0,0.0,1166.3,1335.8,0.0,137.1,0.0
-project_testing-0001.osw,40.0,0.0,16.4,1.5,0.0,0.0,0.0,250.5,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,43.5,-3837.1,11.0,0.0,0.0,0.0,159.2,0.0,2731.3,8194.0,0.0,0.0,0.0,3.0,63.0,27161.4,101666.8,0.0,true,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,1760.6,0.0,3053.7,92.7,346.9,0.0,-3355.0,1749.2,0.0,0.0,1760.6,0.0,5235.4,5614.8,1745.1,0.0,0.0
+project_national-0009.osw,20.0,0.0,5.5,0.0,33.4,18.7,42.9,266.5,0.0,0.0,26.8,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,375.2,1701.3,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,480.4,0.0,0.0,0.0,1220.0,1220.0,1220.0,0.0,0.0,1.0,0.0,3532.9,6015.3,0.0,true,15.2,15.2,0.0,0.0,103.2,0.0,0.0,0.0,102.1,0.0,0.0,0.0,0.0,1364.0,12.1,20.5,30.0,2961.7,33.7,0.0,235.6,0.0,0.0,1166.3,1335.8,0.0,171.3,0.0
```

Comment:
@shorowit Looks like some very big differences in here. Is that expected?

Reply:
I think that's based on how you resolved the merge conflict. The original change only resulted in [small differences](https://github.com/NREL/resstock/pull/635/commits/c52e7f3f2150def3476081ea7469a8eb30296503). Probably it will revert to that once the CI commits again after your merge.